### ISSUE TITLE
Work through list of usual suspect READMEs

### ISSUE THREAD

**Issue by sourishkrout** [stateful/vscode-runme#54]
- [x] https://github.com/stateful/runme.dev/blob/main/README.md
- [x] https://github.com/stateful/vscode-runme/blob/main/examples/README.md
- [x] https://github.com/stateful/vscode-runme/blob/main/examples/vercel/README.md
- [x] https://github.com/stateful/vscode-runme/blob/main/examples/k8s/README.md

Look into automation: https://github.com/stateful/vscode-runme/issues/55 & https://github.com/stateful/runme/issues/58

## Readme Rollcall for
- [x] @christian-bromann: https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md
- [x] @sourishkrout: https://github.com/GoogleCloudPlatform/microservices-demo
- [x] @admc: https://github.com/appium/appium
- [x] @adambabik: https://github.com/micro/micro/

**Comment by adambabik** [comment]
Guessing the language fails for https://github.com/micro/micro/blob/41203a045376fa15aa4ac29cedf69e6f2634a39e/README.md.

**Comment by sourishkrout** [comment]
> Guessing the language fails for https://github.com/micro/micro/blob/41203a045376fa15aa4ac29cedf69e6f2634a39e/README.md.

Little more specific please @adambabik?

**Comment by adambabik** [comment]
@sourishkrout I found the issue:
```
--- a/src/extension/languages.ts
+++ b/src/extension/languages.ts
@@ -61,6 +61,9 @@ export default class Languages {
 
   // todo(sebastian): too naive; should really consider probabilities
   public static weighted(platform: string, results: ModelResult[]): string | undefined {
+    if (!Array.isArray(results) || !results.length) {
+      return
+    }
```

**Comment by sourishkrout** [comment]
Great. What didn't work though? The readme's fairly long

**Comment by adambabik** [comment]
@sourishkrout I didn't pin-point it to a specific code snippet yet. I will try to isolate it and paste it here.

**Comment by sourishkrout** [comment]
> @sourishkrout I didn't pin-point it to a specific code snippet yet. I will try to isolate it and paste it here.

Thank you!

**Comment by sourishkrout** [comment]
Done. Released v0.3: https://github.com/stateful/vscode-runme/releases/tag/0.3.0
